Apromore-Custom-Plugins/Calendar-Portal/src/main/java/org/apromore/plugin/portal/calendar/controllers/Calendars.java

```diff
@@ -390,8 +387,11 @@ public void onClickRestoreBtn() {
         Notification.info(infoText);
     }
 
-    private void updateApplyCalendarButton() {
+    private void updateButtons() {
         applyCalendarBtn.setDisabled(calendarListbox.getSelectedCount() <= 0 || !canEdit);
+        restoreBtn.setDisabled(
+            !canEdit || appliedCalendarId == null || calendarService.getCalendars().stream().noneMatch(c -> c.getId().equals(appliedCalendarId))
```

Comment:
Build is failing due to the following checkstyle violation:
Line is longer than 120 characters (found 144). (393:0) [LineLength]